libs/deps-api/src/lib/deps-api.ts

```diff
@@ -0,0 +1,55 @@
+import { PrismaClient } from '@prisma/client';
+
+import { getQueryParams } from '@verity/utils';
+
+const prisma = new PrismaClient();
+
+export const getAppDepsHandler = async (request: Request) => {
+  const { app, version } = getQueryParams(request);
```

Comment:
Let's try `nextUrl` to get search params

https://github.com/vercel/next.js/discussions/47072#discussioncomment-5291376